src/renderer/library/components/catalog/Catalog.tsx

```diff
@@ -69,6 +69,7 @@ class Catalog extends React.Component<IProps, undefined> {
         const secondaryHeader = <span style={{ display: "flex", justifyContent: "end", alignItems: "end", height: "53px", borderBottom: "1px solid var(--color-verylight-grey-alt)", paddingBottom: "30px" }}><PublicationAddButton /></span>;
 
         const customizationProfileProvision = this.props.customizationProvision.find(({id}) => this.props.customizationProfileId === id);
+        const profileTitle = customizationProfileProvision?.title && typeof customizationProfileProvision.title === "object" ? customizationProfileProvision.title[this.props.locale] || customizationProfileProvision.title["en"] || "" : "";
```

Comment:
Should the fallback string not be empty but `this.props.__("catalog.customization.publications")` instead? (I haven't tested the code, just reading / reviewing it...so I could be totally wrong, sorry :)